api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -478,9 +478,8 @@ message CertificateValidationContext {
   bool only_verify_leaf_cert_crl = 14;
 
   // Config for the max number of intermediate certificates in chain that are parsed during verification.
-  // This does not include the leaf certificate. If configured, and the certificate chain is longer than allowed, the certificates
-  // above the limit are ignored, and certificate validation will fail. The default limit is 100,
-  // though this can be system-dependent.
+  // This does not include the leaf certificate. If configured, certificate chain must reach a trusted issuer no longer than allowed.
+  // Otherwise, certificate validation will fail. The default limit is 100, though this can be system-dependent.
```

Comment:
Maybe something like the following will be better:
```
// Defines the max number of intermediate certificates in the chain that are parsed during verification.
// This does not include the leaf certificate. If a trusted issuer appears in the chain, but in a depth larger than configured, the certificate validation will fail.
// The default limit is 100, though this can be system-dependent. See
// `SSL set_verify_depth
// <https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_set_verify_depth.html>`_.
```

Reply:
Thanks for refine the words, this is correct.

Reply:
I think this comment wasn't updated, maybe missed a commit.
Please let me know if I'm missing anything.

Reply:
@adisuissa Whoops.. I forgot it, will update in next change.

Reply:
done